packages/common/src/enums.ts

```diff
@@ -27,6 +27,7 @@ export enum AUTO_GENERATE_ATTRIBUTE_STRATEGY {
   KSUID = 'KSUID',
   EPOCH_DATE = 'EPOCH_DATE',
   ISO_DATE = 'ISO_DATE',
+  EPOCH_MILLIS = 'EPOCH_MILLIS',
```

Comment:
I would be keen to rename this to `EPOCH_DATE_MILLIS`, to align it with other epoch naming.

But I'll leave this to you.

Happy either way.

Reply:
I don't mind changing the name here, though I'd personally want to avoid `DATE`, despite the name of they Javascript class. Perhaps `EPOCH_TIMESTAMP_MILLIS`, or `EPOCH_INSTANT_MILLIS`?